cfme/tests/cloud/test_providers.py

```diff
@@ -1314,9 +1314,9 @@ def test_add_ec2_provider_with_instance_without_name():
     pass
 
 
+@pytest.mark.provider([EC2Provider], scope="function", override=True, selector=ONE)
```

Comment:
`override=True` should be unnecessary, can you check if this test case works without it?

Reply:
@mmojzis if you check cfme.log, its warning you against using this ;)

Reply:
Removed `override=True`.